index.yaml

```diff
@@ -101,6 +101,13 @@ indexes:
   - name: last_updated
     direction: desc
 
+- kind: FeedbackMessageModel
```

Comment:
Oh! If that's the case, don't remove it. It's probably needed. If you are missing an important index, production errors will occur.

GAE autogenerates index definitions when you make a query in the devserver that requires such an index.